native/README.md

```diff
@@ -0,0 +1,40 @@
+### Building native app with nativefier
+
+You can also build a native version of the app that can run without browser (as a standalone app on your desktop).
+For this purpose we are using [https://github.com/jiahaog/nativefier](nativefier) a tool that transforms the web app into a native desktop application.
+
+#### Get started :
+1. Install nativefier npm package
+```
+npm install -g nativefier
+```
+(you may need to run as `sudo` if you are configuring nativefier for all users)
+
+2. Run conversion 
+```
+nativefier https://chate2ee.fun/  
```

Comment:
how about `npx nativefier https://chate2ee.fun/`
Looks like one-time use, so no need to install it.

Reply:
Yes! forgot that :P. Added.